### PR TITLE
Lesson 2 Level 7 done

### PR DESCRIPTION
**Pull request by kskaretnikov**

### CODE REVIEW

File: src/main/java/students/student_konstantin_karetnikov/Homework/Hello_World.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package students.student_konstantin_karetnikov;
+package students.student_konstantin_karetnikov.Homework;
 
 public class Hello_World {public static void main(String[] args) {System.out.println("Hello World!");
```
Comment (antdolg1) [inline src/main/java/students/student_konstantin_karetnikov/Homework/Hello_World.java:3]: Format code.

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Format code.

**Comment by kskaretnikov** [comment]
formated

File: src/main/java/students/student_konstantin_karetnikov/Homework/Hello_World.java (hunk 1 above)
Comment (antdolg1) [reply src/main/java/students/student_konstantin_karetnikov/Homework/Hello_World.java]: Apply format lines Ctrl+Alt+L

File: src/main/java/students/student_konstantin_karetnikov/Homework/HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package students.student_konstantin_karetnikov.Homework;
+
+public class HelloWorld {public static void main(String[] args)
+{System.out.println("Hello World!");
+}
+}
```
Comment (antdolg1) [inline src/main/java/students/student_konstantin_karetnikov/Homework/HelloWorld.java:6]: Still not formatted. Formatted code should look like this:

```
public class HelloWorld {
    public static void main(String[] args) {
        System.out.println("Hello World!");
    }
}
```

**Review by antdolg1 — APPROVED** [review]

**Pull request merged**